includes/settings/settings-ppec.php

```diff
@@ -45,9 +45,15 @@
 
 $credit_enabled_label = __( 'Enable PayPal Credit', 'woocommerce-gateway-paypal-express-checkout' );
 if ( ! wc_gateway_ppec_is_credit_supported() ) {
-	$credit_enabled_label .= '<p><em>' . __( 'This option is disabled. Currently PayPal Credit only available for U.S. merchants.', 'woocommerce-gateway-paypal-express-checkout' ) . '</em></p>';
+	$credit_enabled_label .= '<p><em>' . __( 'This option is disabled. Currently PayPal Credit only available for merchants using USD currency.', 'woocommerce-gateway-paypal-express-checkout' ) . '</em></p>';
 }
 
+$location               = wc_get_base_location();
+$credit_enabled_default = 'US' === $location['country'] ? 'yes' : 'no';
+
+$credit_enabled_description  = __( 'This enables PayPal Credit, which displays a PayPal Credit button next to the Express Checkout button. PayPal Express Checkout lets you give customers access to financing through PayPal Credit® - at no additional cost to you. You get paid up front, even though customers have more time to pay. A pre-integrated payment button shows up next to the PayPal Button, and lets customers pay quickly with PayPal Credit®.', 'woocommerce-gateway-paypal-express-checkout' );
+$credit_enabled_description .= __( ' (Should be unchecked for stores involved in Real Money Gaming.)', 'woocommerce-gateway-paypal-express-checkout' );
```

Comment:
Concatenation will break RTL translations.

Reply:
Oh right – will just modify the string.

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/pull/425/files/a7e1143d4b5113361a87eae9f5ba39acb6039a19..40a07037f36eeb7cb06eb844e3b499111dd70e3f